Add environment as a configuration option

heytrav opened the pull request:
Sentry now supports displaying deploy environment information (i.e. staging,
production, etc.). This adds it as either a CLI option or an environment
variable SENTRY_ENVIRONMENT.

Commit:
Add environment as a configuration option

Sentry now supports displaying deploy environment information (i.e. staging,
production, etc.). This adds it as either a CLI option or an environment
variable SENTRY_ENVIRONMENT.

heytrav commented:
Hi @imankulov, please let me know if this patch is ok. We're testing the environment tag in our project at the moment and are keen to get this deployed.

imankulov commented:
Hi @heytrav 

Thanks for the pull request!

According to the [CHANGELOG](https://github.com/getsentry/raven-python/blob/master/CHANGES#L153), the environment parameter appeared in v5.9.0 around December 2015 and I'm afraid it may break older installations. Can you please modify your patch and correct the dependency in `setup.py`

Commit:
Pin to raven version where environment was added

In v5.9.0 Sentry added the "environment" parameter. Pin to this version in
order to not break older installations.

heytrav commented:
@imankulov nice catch! Pinned the change in `setup.py` and pushed.

imankulov commented:
Thanks a lot for your updates. New version released and available on PyPI
